cmd/zpool/zpool_main.c

```diff
@@ -3736,6 +3736,12 @@ print_vdev_stats(zpool_handle_t *zhp, const char *name, nvlist_t *oldnv,
 	    &oldchild, &c) != 0)
 		return (ret);
 
+	/*
+	 * Use MIN() here to deal with VDEVs being added while we're viewing
+	 * the stats.
+	 */
+	children = MIN(c, children);
```

Comment:
Rather that continuing to borrow `c` for this check why not add a new `oldchildren` variable for clarity.  Then the `MIN` check would be clear without the need for any comment.